Sources/CvPlayer.cpp

```diff
@@ -21933,7 +21905,7 @@ void CvPlayer::read(FDataStreamBase* pStream)
 	// @SAVEBREAK REPLACE
 		double fGold = 0;
 		WRAPPER_READ(wrapper, "CvPlayer", &fGold);
```

Comment:
don't need these brackets?

Reply:
what brackets?

Reply:
this->m_iGold = static_cast<int64_t>(fGold + 0.01) + (1000000 * m_iGreaterGold) + m_iGold;
		this->m_iGold = static_cast<int64_t>(fGold + 0.01) + 1000000 * m_iGreaterGold + m_iGold;

Reply:
They are not needed. multiplication always happen before addition.

Reply:
ha

Reply:
i.e. it would be needed if it was like so:
`this->m_iGold = static_cast<int64_t>(fGold + 0.01) * (1000000 + m_iGreaterGold) * m_iGold;`
Because this would not be the same
`this->m_iGold = static_cast<int64_t>(fGold + 0.01) * 1000000 + m_iGreaterGold * m_iGold;`